### ISSUE TITLE
It is in cache but does not show image

### ISSUE THREAD

**Issue by muhammetdemirci** [wcandillon/react-native-img-cache#10]
<img width="783" alt="screen shot 2017-02-20 at 17 12 16" src="https://cloud.githubusercontent.com/assets/21204536/23128335/41e33f98-f787-11e6-9e7d-f51701928d12.png">

<img width="768" alt="screen shot 2017-02-20 at 17 07 48" src="https://cloud.githubusercontent.com/assets/21204536/23128316/2caae14e-f787-11e6-8fb9-cd6d7112556c.png">

**Comment by wcandillon** [comment]
@muhammetdemirci Just out of curiosity, how did you get the inspector screenshot? It looks like a really useful tool I could use.

Are you positive that the issue is not related to the styles applied the elements? For instance if `CachedImage` has no size, it won't be displayed. Is it a regression from `Image` If yes, I can look into it.

**Comment by muhammetdemirci** [comment]
I use react-devtools 


and my item.js 
<img width="585" alt="screen shot 2017-02-20 at 18 13 32" src="https://cloud.githubusercontent.com/assets/21204536/23130629/dd7aa2b8-f78f-11e6-8879-e231984b717f.png">
<img width="812" alt="screen shot 2017-02-20 at 18 13 43" src="https://cloud.githubusercontent.com/assets/21204536/23130631/ddafcdc6-f78f-11e6-9191-e3cb7e4909a8.png">
<img width="817" alt="screen shot 2017-02-20 at 18 13 51" src="https://cloud.githubusercontent.com/assets/21204536/23130630/ddaed164-f78f-11e6-9a49-86f2c654dc3e.png">

**Comment by wcandillon** [comment]
@muhammetdemirci thks for the tip!

If you replace `CachedImage` with `Image`, then the image displays? Could it be because now the `Image` component to which the style is applied is not a direct children of `View`? If this is the case, I would need to investigate how this can be solved.

**Comment by muhammetdemirci** [comment]
When I use Image ,unfortunately the image doesn't display

**Comment by wcandillon** [comment]
@muhammetdemirci So you are able to reproduce the issue without using `CachedImage`? Have you tried to set a `width` and `height` to display the image?

**Comment by muhammetdemirci** [comment]
I set a width and height with using Image , but it also didn't display

**Comment by smkhalsa** [comment]
I'm having a similar issue.

**Comment by wcandillon** [comment]
@muhammetdemirci As far as I understand, this is a styling issue no? In fact you are having the same issue without the `CachedImage` component? Is my understanding correct?

**Comment by smkhalsa** [comment]
For me my images display correctly with the standard Image component, but if I swap this out for a CachedImage component, it's blank.

**Comment by wcandillon** [comment]
@smkhalsa can you share an example. Especially the layout/styling in which you use `CachedImage`? Is there a `width` and `height` set to the component?

**Comment by movie4** [comment]
Hello, got same issue image not displayed

rn: 41
android: 6
macOS/Linux

```
image: {
    width: 70,
    height: 70,
    resizeMode: 'contain',
  },
```

```
return (
  <View>
    <CachedImage style={styles.image} source={{ uri: 'https://avatars3.githubusercontent.com/u/1005998?v=3&s=88' }} mutable />
    <Image style={styles.image} source={{ uri: 'https://avatars3.githubusercontent.com/u/1005998?v=3&s=88' }} />
  </View>
)
```

**Comment by movie4** [comment]
maybe this help: https://github.com/wkh237/react-native-fetch-blob/issues/215

**Comment by wcandillon** [comment]
@movie4 Thanks a lot for pointing me to this issue, I will fix it right away.

**Comment by wcandillon** [comment]
@movie4 @smkhalsa Can you check if the issue is fixed with `1.1.1`? Feel free to reopen otherwise.

**Comment by wcandillon** [comment]
@movie4 @smkhalsa Please also make sure that in `AndroidManifest.xml` the following permission is set:
```xml
<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" /> 
```

**Comment by movie4** [comment]
@wcandillon test on android 6 and 7

not working - https://www.youtube.com/watch?v=YKy2Z_JybFg 
use - react-native-img-cache@1.1.4

**Comment by wcandillon** [comment]
@movie4

**Comment by wcandillon** [comment]
@movie4 yes still doesn't work for me on android, I'm looking into it.

**Comment by wcandillon** [comment]
@movie4 @smkhalsa @muhammetdemirci Finally, the android support has been fixed in `1.1.6`. Thank you for your help guys!

**Comment by movie4** [comment]
it work!  many thanks @wcandillon
